models/net.py

```diff
@@ -149,99 +141,75 @@ def __init__(
         Args:
             patchmatch_interval_scale: depth interval scale in patchmatch module
             propagation_range: propagation range
-            patchmatch_iteration: patchmatch interation number
+            patchmatch_iteration: patchmatch iteration number
             patchmatch_num_sample: patchmatch number of samples
-            propagate_neighbors: number of propagation neigbors
-            evaluate_neighbors: number of propagation neigbors for evaluation
+            propagate_neighbors: number of propagation neighbors
+            evaluate_neighbors: number of propagation neighbors for evaluation
         """
         super(PatchmatchNet, self).__init__()
 
         self.stages = 4
         self.feature = FeatureNet()
         self.patchmatch_num_sample = patchmatch_num_sample
 
-        num_features = [8, 16, 32, 64]
+        num_features = [16, 32, 64]
 
         self.propagate_neighbors = propagate_neighbors
         self.evaluate_neighbors = evaluate_neighbors
         # number of groups for group-wise correlation
         self.G = [4, 8, 8]
 
         for i in range(self.stages - 1):
-
-            if i == 2:
-                patchmatch = PatchMatch(
-                    random_initialization=True,

```

Comment:
No need for this flag anymore. Random initialization will happen the first time around be checking that the tensor is empty.